src/resize-observer.d.ts

```diff
@@ -0,0 +1,244 @@
+/**
```

Comment:
Any chances to dev-depend on [`@types/resize-observer-browser@0.1.4`](https://unpkg.com/browse/@types/resize-observer-browser@0.1.4/) instead? I see that `device-pixel-content-box` value [is missing here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/0a8d7131547bd2ca4c491f5d820979a8da6c4863/types/resize-observer-browser/index.d.ts#L20), but... if that's the only problem we have, why not use some typing workaround for a while, before these typings are fixed? Even `devicePixelContentBoxSize` property is already available.

Reply:
Decided to leave this typings as-is, and to make a PR to DefinitelyTyped repo.